ironfish-cli/src/commands/deposit.ts

```diff
@@ -112,20 +117,23 @@ export default class Bank extends IronfishCommand {
 
     const balanceResp = await this.client.getAccountBalance({ account: accountName })
     const confirmedBalance = Number(balanceResp.content.confirmed)
-    const requiredBalance = ironToOre(IRON_TO_SEND) + fee
 
-    if (confirmedBalance < requiredBalance) {
-      this.log(`Insufficient balance: ${confirmedBalance}. Required: ${requiredBalance}`)
+    if (confirmedBalance < fee + ironToOre(minDeposit)) {
+      this.log(
+        `Insufficient balance: ${confirmedBalance} ORE. At minimum, depositing requires the fee (${fee} ORE) plus a minimum deposit (${minDeposit} IRON)`,
+      )
       this.exit(1)
     }
+    const sendableIron = oreToIron(confirmedBalance - fee)
+    const ironToSend = sendableIron < maxDeposit ? sendableIron : maxDeposit
```

Comment:
Is it acceptable to default to spendable balance (total wallet balance - fee)? Or should we be doing something else?